src/components/textbox/textbox.stories.tsx

```diff
@@ -28,11 +29,71 @@ export const CharacterCounter: ComponentStory<typeof Textbox> = () => {
       onChange={setValue}
       characterLimit="10"
       enforceCharacterLimit={false}
-      warnOverLimit
     />
   );
 };
 
+export const CharacterCounterTranslations: ComponentStory<
+  typeof Textbox
+> = () => {
+  const [state, setState] = useState("Textbox");
+  const setValue = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    setState(target.value);
+  };
+  return (
+    <>
+      <I18nProvider
```

Comment:
Same as in Textarea stories, the english translations override example is kinda redundant.